test/ProductionModePlugin.js

```diff
@@ -1,37 +1,55 @@
-var expect = require("chai").expect;
 var GlobalizePlugin = require("../index");
-var PathChunkPlugin = require("path-chunk-webpack-plugin");
+var expect = require("chai").expect;
+var fs = require("fs");
+var mkdirp = require("mkdirp");
 var path = require("path");
-var webpack = require("webpack");
 var rimraf = require("rimraf");
-var fs = require("fs");
+var webpack = require("webpack");
```

Comment:
Done. Added a rule for [sort-requires](https://www.npmjs.com/package/eslint-plugin-sort-requires) to this PR